Add Promtail Limit Stage to Grafana Agent Operator (#2741)
grafana/agent: docs/sources/operator/api.md

</tr>
</tbody>
</table>
<h3 id="monitoring.grafana.com/v1alpha1.LimitStageSpec">LimitStageSpec
</h3>
<p>
(<em>Appears on: </em><a href="#monitoring.grafana.com/v1alpha1.PipelineStageSpec">PipelineStageSpec</a>)
</p>
<div>
<p>The limit stage is a rate-limiting stage that throttles logs based on
several options.</p>
</div>
<table>
<thead>
<tr>
<th>Field</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<code>rate</code><br/>
<em>
int
</em>
</td>
<td>
<p>The rate limit in lines per second that Promtail will push to Loki.</p>
</td>
</tr>
<tr>
<td>
<code>burst</code><br/>
<em>
int
</em>
</td>
<td>
<p>The cap in the quantity of burst lines that Promtail will push to Loki.</p>
</td>
</tr>
<tr>
<td>
<code>drop</code><br/>
<em>
bool
</em>
</td>
<td>

> <p>When drop is true, log lines that exceed the current rate limit will be discarded.
> When drop is false, log lines that exceed the current rate limit will only wait
> to enter the back pressure mode.</p>
> <p>Defaults to false.</p>

Reviewer: ```suggestion
<p>When drop is true, log lines that exceed the current rate limit are discarded.
When drop is false, log lines that exceed the current rate limit wait
to enter the back pressure mode.</p>
<p>Defaults to false.</p>
```
